include/ews/rapidxml/rapidxml.hpp

```diff
@@ -91,10 +91,20 @@ namespace rapidxml
         {
         }
 
+        // TODO make what() noexcept
```

Comment:
This is not really relevant for EWS, no ticket needed. Just a small TODO for some unspecified point in the near future